synapse/handlers/message.py

```diff
@@ -446,7 +439,7 @@ async def create_event(
         event_dict: dict,
         token_id: Optional[str] = None,
         txn_id: Optional[str] = None,
-        prev_event_ids: Optional[Collection[str]] = None,
+        prev_event_ids: Optional[List[str]] = None,
```

Comment:
There's a bunch of changes of `Collection` -> `List` which I think is actually necessary because this eventually gets serialized to JSON?